frontend/src/index.tsx

```diff
@@ -38,16 +38,28 @@ const MobXAwareRouter: FunctionComponent = (props) => {
     );
 };
 
+export const FakeRoot: React.FunctionComponent<{
+    children: React.ReactNode;
+    appState?: AppState;
+}> = ({appState, children}) => {
+    const inner = () => <MobXAwareRouter>
```

Comment:
No need for a function
```suggestion
    const inner = <MobXAwareRouter>
```